### ISSUE TITLE
The default voltage warning level in 1.2.02 is dangerously low

### ISSUE THREAD

**Issue by choochoo2** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#180]
The default warning level for the X10 with V1.2.02 is 6.4V.  Testing shows that by the time voltage drops to that warning level it is already in a steep dive, and in a couple of minutes your plane will be too.  An individual user may choose to cut things this close but the default level should be more conservative and allow users at least several minutes to heed the warning and land their aircraft.  It would appear that a warning level of 6.8V would give full use of the battery and still allow users plenty of time to get the plane on the ground before the tx voltage drops to un-flyable levels.

Along the same lines, the default values for the display icon do not adequately inform the user when the battery is approaching exhaustion.  I haven't yet worked out values to recommend but the default values were showing the battery as half full shortly before the tx issued a low voltage warning.

**Comment by Woita2095** [comment]
Agree. I haven't tested it at my X12S (maybe I will). Most of users are flying with default values, it should be more conservative, as choochoo2 said.
Agree. I do not test it at my X12S (maybe I will). Most of users are flying with default values, it should be more conservative, as choochoo2 said.
